algorithmic-strategies/divide-and-conquer.md

```diff
@@ -0,0 +1,292 @@
+# Divide & Conquer
+
+## Learning Goals
+
+By the end of this lesson we should be able to:
+
+- Explain what a divide-and-conquer solution is
+- Write some divide-and-conquer solutions
```

Comment:
I made a different straightforward suggestion above, but here's another one...

I think that this topic could use some framing about _why_ students should learn this topic, now. How is it useful? Why should they pay attention?

This is an attempt to hit that goal, and to rewrite this section

```suggestion
## Goals

As developers, we need as many tools to solve problems as possible.

When we understand the _kind_ of solution or algorithm we are using or can use, we can more easily read and understand it.

There are a number of solutions and algorithms classified as using the Divide and Conquer approach. Learning about Divide and Conquer approaches will:

- Help us identify divide and conquer algorithms
- Help us think about, design, and write divide and conquer solutions
```

Reply:
I like this approach better, especially after you mentioned adding something like this while doing the dynamic programming review. I just updated the casing on divide-and-conquer approach.